lib/traject/dor_config.rb

```diff
@@ -95,7 +95,7 @@
 to_field 'url_suppl', stanford_mods(:term_values, [:related_item, :location, :url])
 
 # publication fields
-to_field 'pub_search', stanford_mods(:place)
+to_field %w(pub_display pub_search), stanford_mods(:place)
```

Comment:
We want pub_display to be searchable, and as it stands, this would result in some circular logic.

Currently:

pub_search is copied to pub_display. In other words, pub_display is always pub_display's original values plus whatever is in pub_search.
pub_search is copied to all_search.

If we want pub_display to be searchable, we could copy it over pub_search. But pub_search is already copied over to pub_display.

If we want pub_display and pub_search to be equally searchable, it seems more consistent to make them equivalent from the beginning and remove the copy field in the schema.